components/script/dom/globalscope.rs

```diff
@@ -2529,7 +2537,7 @@ impl GlobalScope {
 
     /// Evaluate JS code on this global scope.
     pub fn evaluate_js_on_global_with_result(&self, code: &str, rval: MutableHandleValue) -> bool {
-        self.evaluate_script_on_global_with_result(code, "", rval, 1)
+        self.evaluate_script_on_global_with_result(code, "", rval, 1, None)
```

Comment:
Yes! I think it would be good to always pass a URL but one of the reason why I made it an Option is, if an user tries to import a module from devtools, it might not have a URL?

Or, let's just use `global.api_base_url()` in that case 👀? (which might make more sense!)

Reply:
I'm inclined to make the argument non-optional and make every caller decide.

Reply:
I see! Taking non-optional makes sense to me! I will fix it! Thanks!

Reply:
It's fixed in 870692c 

(Also see https://github.com/servo/servo/pull/27026#discussion_r454778062)